commands/combat.py

```diff
@@ -324,10 +324,10 @@ def func(self):
             combat_log_entry(caller, combat_string)
 
         # "Tick" to have a round pass.
-        combat_tick(caller)
+        negative_lf_from_poison = combat_tick(caller)
```

Comment:
Not sure if it makes sense to have a return value from a function called `combat_tick` be a poison damage number. It does feel like it makes sense to leave combat_tick as the function that actually does all of the ticking modification